src/services/viaplay/ViaplayApi.ts

```diff
@@ -94,11 +119,13 @@ export interface ViaplayProductUserInfo {
 }
 
 class _ViaplayApi extends ServiceApi {
-	INITIAL_URL = 'https://viaplay.com/requirements';
+	INITIAL_URL = 'https://viaplay.com/';
```

Comment:
While this may be no longer be correct, it was introduced because of #148
Not sure why you change it, but  perhaps it could be updated to something not broken, like https://viaplay.com/devices ?

Reply:
The /requirements url was pointing to a 404, which is why I changed it. That may have been a mistake though, as some regions may require a path for the login to work? Let's see if any users experience issues with the change.